cohorts/load.py

```diff
@@ -213,6 +213,14 @@ class Cohort(Collection):
     responder_pfs_equals_os : bool
         Ensure that the PFS values for responders (not progressed) are equal to
         OS values.
+    check_provenance : bool
+        Verify that the cached provenance is equal to the current environment.
+    print_provenance : bool
+        Print a cached provenance summary.
```

Comment:
Was supposed to be read as (cached provenance) summary vs. cached (provenance summary), but I'll update :)